contents/plenum/2020-10-20/_index/main.md

```diff
@@ -23,6 +23,8 @@
 4. On an accrual basis we are up/down $?/month ?
     - **including predicted upcoming costs amortised monthly, we are _down_ $620 a month**
 
+- (Valentine) request for expense breakdown by category, on an annualised (365/12) basis.
```

Comment:
@dyanawu For your action.

Reply:
Regular Expenses | Amount | Frequency | Monthly Cost (amortised)
-- | -- | -- | --
Rental | 3400 | 12 | $ 3,400.00
Utilities | 450 | 12 | $ 450.00
Internet | 148 | 12 | $ 148.00
CorpSec | 240 | 2 | $ 40.00
Accounting | 3000 | 1 | $ 250.00
Aircon | 980 | 1 | $ 81.67
Cleaning | 80 | 26 | $ 173.34
Income Tax | 600 | 1 | $ 50.00
  |   |   | $ 4,593.01